include/oneapi/tbb/task_arena.h

```diff
@@ -640,6 +642,18 @@ inline void end_parallel_phase(bool with_fast_leave) {
 }
 #endif
 
+inline std::vector<d1::task_arena> create_numa_task_arenas(d1::constraints c = {},
+                                                           unsigned reserved_slots = 0)
+{
+    static std::vector<numa_node_id> node_indices = d1::numa_nodes();
+    std::vector<d1::task_arena> numa_arenas;
+    numa_arenas.reserve(node_indices.size());
+    for (auto numa_id : node_indices) {
+      numa_arenas.emplace_back(c.set_numa_id(numa_id), reserved_slots);
```

Comment:
As far as I can see, the indentation in this file adheres to four spaces where applicable.
```suggestion
        numa_arenas.emplace_back(c.set_numa_id(numa_id), reserved_slots);
```